tests/models/tfliteModels/depthwise_conv2d_c1_m2.inp0

```diff
@@ -0,0 +1 @@
+?�7?�N?w}?H��>QY%?n�><Kd?��v?rR�>~�J?�e?^k?��l?Z{�=�p�=
��<�&U?H5G?�^?��z?��L?G�>��G?�9�=
```

Comment:
Please mark these files as binary to avoid non-utf8 text warnings. 

https://stackoverflow.com/questions/7994811/git-mark-file-as-binary-to-avoid-line-separator-conversion

Reply:
Is this ok (all the files in "tfliteModels" directory are binaries including the models themselves)?
```
* binary
```

Reply:
The goal is when you update the PR, all the inp0 and model files should show up as binary. For example, in current status some are:
tests/models/tfliteModels/greater.inp0

But tests/models/tfliteModels/depthwise_conv2d_c1_m2.inp0 is not.

Reply:
And where do I look to see whether the files are marked as binary or not?

Reply:
I have no clue how to do it.

Reply:
@mciprian13, I need to confirm this is the issue on these files still, but I think the issue here is that both git and our internal linters use presence of NUL to determine if a file is binary or not and for the files that aren't marked as binary probably don't contain a NUL. Changing the git config could make git realize these are binary but won't help with our linters. Is there alternative way we can encode or otherwise deal with these files to fix this?

Reply:
@jackm321 @yinghai I modified the file dumps to include a leading NULL byte (which of course is skipped when loading the data in the importer tests). Right now GIT recognizes all the files as being binary. Hopefully same thing happens in your internal linters.
Let's have this imported and hopefully landed (the OpenCL seems to fail for some reason not related to this PR).
Thanks!

Reply:
@mciprian13 that has fixed it, thanks for fixing that!